pages.es/linux/usermod.md

```diff
@@ -1,12 +1,21 @@
 # usermod
 
 > Modifica una cuenta de usuario.
+> Ver también: `users`, `useradd`, `userdel`.
 > Más información: <https://manned.org/usermod>.
 
 - Cambia el nombre de un usuario:
 
 `usermod -l {{nuevo_nombre}} {{usuario}}`
 
+- Cambia el ID de un usuario:
+
+`sudo usermod --uid {{id}} {{username}}`
+
+- Cambia la interfaz de comandos(shell) a un usuario:
+
+`sudo usermod --shell {{path/to/shell}} {{username}}`
+
 - Añade un usuario a grupos suplementarios (tener en cuenta los espacios en blanco):
```

Comment:
```suggestion
- Añade un usuario a grupos suplementarios (ten en cuenta los espacios en blanco):
```

Reply:
Related to a previous comment, let me know if we use "tenga" or "ten" to sync.

Reply:
ten